spec/models/shelter_spec.rb

```diff
@@ -67,5 +80,11 @@
         expect(@shelter_1.pet_count).to eq(3)
       end
     end
+
+    describe ".order_shelters_reverse_alphabetically" do
+      it "returns shelters in reverse alphabetical order (z to a)" do
+        expect(Shelter.order_shelters_reverse_alphabetically).to eq([@shelter_2, @shelter_3, @shelter_1])
+      end
+    end
```

Comment:
Great work testing at the unit (model spec) and integration (feature spec) level!